cypress/e2e/buy-shirt.cy.ts

```diff
@@ -0,0 +1,26 @@
+describe("Buy a t-shirt", () => {
+  it("then the t-shirt should be bought", () => {
+    cy.visit("http://automationpractice.com/");
+    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+    cy.get(
+        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
+    ).click();
+    cy.get("[style*='display: block;'] .button-container > a").click();
+    cy.get(".cart_navigation span").click();
+
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+
+    cy.get("#SubmitLogin").click();
+    cy.get("#center_column button").click();
```

Comment:
I think you can improve this.
Maybe looking not in the center_column but inside the form or even better inside cart_navigation